common/lib/xmodule/xmodule/tests/test_capa_module.py

```diff
@@ -361,6 +361,108 @@ def test_showanswer_past_due(self):
                                             graceperiod=self.two_day_delta_str)
         self.assertFalse(still_in_grace.answer_available())
 
+    def test_showanswer_after_attempts_with_max(self):
+        '''
+        Button should not be visible when attempts < required attempts.
```

Comment:
Per f2f, force one of these tests to fail, so that we can see how it formats this message/title.
We may need a newline after the first line of text.
Actually, I think best practices actually recommend that, so let's go ahead and do it regardless.

```python
def foo:
    """
    Button should not be visible when attempts < required attempts.

    Even with max attempts set, the show answer button should only
    show up after the user has attempted answering the question for
    the requisite number of times, i.e `attempts_before_showanswer_button`
    """
```